DOCKERFILE

```diff
@@ -0,0 +1,25 @@
+ARG swift_version=5.8
```

Comment:
Let's bump this to `5.10` since we don't have to support earlier versions

Reply:
Updated!